### DISCUSSION TITLE
DJI Android

### DISCUSSION THREAD

**Discussion by bakedtart** [deepjavalibrary/djl#1482] [Q&A]
Hi, under DJL Android section we were asked to include the snapshot repository and 4 modules to our dependencies. 

But from the repository (https://oss.sonatype.org/#nexus-search;quick~ai.djl.android), I do not see the latest version - 0.15.0. May I know where or how should I get the latest version?

Thank you.

**Comment by frankfliu** [comment]
@bakedtart

The latest android package we published is 0.11.0. We will try to prioritize publishing 0.15.0 in a couple of weeks.

**Comment by frankfliu** [comment]
@bakedtart 

There is bug in DJL 0.15.0 core api package that breaks android pytorch library loading. It's being fixed in 0.16.0-SNAPSHOT version. Please use 0.16.0-SNAPSHOT version for now. I have updated the README: https://github.com/deepjavalibrary/djl/tree/master/android#installation

Please see usage in our android demo project for 0.16.0-SNAPSHOT version: https://github.com/deepjavalibrary/djl-demo/tree/master/android/pytorch_android/quickdraw_recognition
